### DISCUSSION TITLE
multi worlds for /tpr

### DISCUSSION THREAD

**Discussion by Leahcimkrob** [EssentialsX/Essentials#4679] [Ideas and Feature Suggestions] [answered]
Is it possible that you expand the random teleport to multiple worlds?

**Comment by pop4959** [comment, marked as answer]
It's not currently possible, but there is work being done in #4271 to support this. Recently bumped it to support 1.18.1 so if you would like to  test it and give your feedback, feel free to do so.

**Reply by Leahcimkrob** [reply]
Green can I test the version, where can I download it?

**Reply by pop4959** [reply]
I left instructions on how to get it in the linked pull request.
